### ISSUE TITLE
Change management for removing Button Styles from CKEditor 5

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15154]
## Description
When we update to D10 and CKEditor 5, the 3 Button styles will no longer be available in the CKEditor. However, we will not be doing any updates or changes to any links that are currently using these styles. That will happen later, after the D10 launch. The only change happening at this time is removing the ability to add these button styles to new links going forward.

## Acceptance Criteria
- [x] Create change management plan
- [x] Product feedback and/or approval
- [x] Document in [GitHub ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/product-outlines/ckeditor/README.md) with other upcoming CKEditor changes
- [x] Notification to Randi and CAIA team for awareness
- [x] Update [existing draft KB article](https://prod.cms.va.gov/node/34036/latest) to include information about Style dropdown and how to use remaining styles

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by BerniXiongA6** [comment]
Note to @BerniXiongA6 @EWashb @laflannery -- let's sync up and chat about this before Sprint 93 refinement. I'll find a time for us 3 and send out an invite.

**Comment by laflannery** [comment]
## Proposed change management plan

Within the CMS, we are removing button styles from the styles dropdown in the rich text editor toolbar.

This means that editors will no longer be able to style links to visually look like buttons. However, this change does NOT change the styling of any existing content. Any existing links that are styled using these buttons styles will continue to use this styling - unless manually and purposefully changed by the editor.

This change is being done to help editors make sure the content they create is inline with the VA Design guidelines and to ensure material honesty within the content. Material honesty means that elements used within content should look like what they are - links should look like links and buttons should look like buttons. These elements have different keyboard interactions and if they look different from what they are, this could cause confusion and frustration for some users by not responding to expected keys.

This will impact all CMS Editors.

This change adheres to VA policies and standards because, as mentioned previously, it is assisting editors in making sure the content they create is accessible and inline with the VA Design guidelines. 

There is an [existing KB article](https://prod.cms.va.gov/help/cms-basics/how-to-use-the-rich-text-editor) regarding the options available in the editor and how to use it. This article will need to be updated to change the "Address Block" section to be a more generic "Styles" section.

This change will not be tested with users prior to launch.

This change will occur at the same time that the update to D10 and CKEditor 5 occurs.


### Updated KB article
https://prod.cms.va.gov/node/34036/latest (I actually needed to make a few modifications for the other changes as well)

* This will need to get published when D10 launches

**Comment by laflannery** [comment]
@BerniXiongA6 and @EWashb Would you be able to review the comment above as well as the update [Github](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/product-outlines/ckeditor/README.md#remove-button-styles-from-style-dropdown-on-toolbar) and let me know if there is anything you feel needs to be changed/added regarding the change management plan here?

**Comment by laflannery** [comment]
@TroyA6 KB article is updated as well (link in comment above)

**Comment by EWashb** [comment]
This looks good to me. I wonder if it would be worth it to send an email blast to editors with all of the accessibility changes occurring in the D10 upgrade. Maybe that's too much for the small number of changes? Just brainstorming. @laflannery

**Comment by BerniXiongA6** [comment]
Food for thought @EWashb -- even with an email blast, we would want to have a well thought-out comms strategy about what that letter might say. We did this on a previous project where we crafted emails at different intervals with the messaging that a change was coming, what those changes were, and who to contact/what to expect, etc.. If we go that route, we would want to have a those comms artifacts vetted and QA'ed before sending it out. It might be too big of a lift to consider doing now, but I wish we had a release notes/what's new page where we could communicate changes and updates for editors. cc: @laflannery

**Comment by BerniXiongA6** [comment]
@laflannery -- there are a couple of tiny suggested edits to the updated KB article that I'll share with you tomorrow. But great job, things look fantastic so far! cc: @EWashb

**Comment by laflannery** [comment]
@BerniXiongA6 and @EWashb I agree that a series of email blasts might be too much right now, we maybe should have started this sooner however to Berni's idea - what about if we created a new KB article for the release notes? Edmund has mentioned this before as something that Drupal could do but since we haven't put this together yet a manual KB article maybe could be a stop gap? I'm thinking just a very basic article calling out important points and linking to anything else, like this other updated article. Just a thought!

**Comment by EWashb** [comment]
Yea, I like that idea. Wish I had thought of something sooner but I like the KB idea @BerniXiongA6 @laflannery

**Comment by BerniXiongA6** [comment]
Love this, thanks for finding this @laflannery ! I say yes, what say you boss @EWashb ?

**Comment by EWashb** [comment]
@laflannery @BerniXiongA6 I was just talking to other POs this morning about updating that exact space for our CM purposes. I think I even have a ticket in the backlog about resuming use of that for our releases. I would say yes, if we can use this CMS announcement page again I would love to. It helps this initiative as well as many others. Other teams can benefit from it too. 

So first, let's revamp it with our D10 use case. Then another issue to create is a runbook of sorts on using this for more change announcements and sharing that info with our product teams. Let me know if you have more questions or thoughts.

**Comment by BerniXiongA6** [comment]
@BerniXiongA6 please create a new ticket for this "cms announcements" work that we can update with D10 launch (include in the AC that we could tee up this announcement in the Drupal CMS Office Hours / if it's ready to tee up). cc: @laflannery @EWashb

**Comment by laflannery** [comment]
This all sounds great so just to close this ticket out @BerniXiongA6 Let me know what the suggestions/updates are for the KB and then I can wrap that up, thanks!

**Comment by BerniXiongA6** [comment]
The sentence (selected in blue color) might have some extraneous words in it, can you double check?

<img width="848" alt="media can only be linked to -- screenshot for Laura" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/85254824/49fd9628-f50e-4064-bfa3-de7815f24b4a">

**Comment by BerniXiongA6** [comment]
@laflannery the "to" in the sentence (selected in blue color) seems extraneous and could probably be removed.
 
<img width="754" alt="action links - screenshot for laura" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/85254824/8ce3d3d2-232a-4e70-94d7-cdfcbd43cf64">

**Comment by BerniXiongA6** [comment]
Suggested edits to puncutation in the second sentence in bullet number 4 could read: "In this example, it would be "Content Management System." 

<img width="852" alt="punctuation error - screenshot for laura" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/85254824/f05de16b-fdd0-4a3d-b1bf-fbb946c651cc">

**Comment by BerniXiongA6** [comment]
@laflannery Suggested edits to first sentence could read as two sentences instead: "It is not recommended that you copy and paste text into the rich text editor from other sources. Examples of other sources include emails, word documents, or other text editors.

<img width="739" alt="copy paste - screenshot for laura" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/85254824/3d23e430-7940-4b91-870c-180148781b12">

**Comment by laflannery** [comment]
@BerniXiongA6 I believe I got all your updates. The first one however wasn't extraneous words, it was referring to a specific content block. I bolded the name of the block (which is typically how we signify field names) and then I added a link to the KB article that explains more about using this and linking to files if the editors needs it. I think that makes this more clear:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/51ea8ba6-51c5-4a0e-b4f9-e06c6d9629d9)

I'm going to close this but let me know if this needs needs any other updates!
